package.json

```
@@ -34,7 +34,7 @@
     "rollup": "^2.79.1",
     "rollup-plugin-sourcemaps": "^0.6.3",
     "ts-node": "^10.9.1",
-    "typescript": "^3.9.10"
+    "typescript": "^4.9.4"
```

Could you check @voxpelli's comment?
https://github.com/eslint-community/regexpp/pull/15#discussion_r1058435149

```suggestion
    "typescript": "~4.9.3"
```